test/app.spec.js

```diff
@@ -686,4 +686,49 @@ describe('app', () => {
         })
     );
   });
+
+  describe('emitOptionalEvents', () => {
```

Comment:
The test for the methods don't match the order they show up in `Messenger`. Readers shouldn't have to jump around files as they scroll to match test cases to the source. These two tests should go between the test cases for `onPostback` and `send`